ttnn/cpp/ttnn/tensor/tensor.cpp

```diff
@@ -105,6 +106,7 @@ Tensor::~Tensor() {
 
 void Tensor::deallocate(bool force) {
     ZoneScopedN("TensorDeallocate");
+    GraphTracker::instance().track_begin_op("Tensor::deallocate", force);
```

Comment:
these manual instrumentations will eventually get moved to higher level and these functions will get wrapped, right? This is what you meant in today's sync?

Reply:
That what I discussed with you. Thats why I want to fix original tensor asap. But in other pr.